### DISCUSSION TITLE
Udacity nanodegree

### DISCUSSION THREAD

**Discussion by sohels07** [community/community#22923] [New to GitHub]
<p>Has udacity removed the 1-month-free trial for students using github’s student development package for nanodegree courses?</p>

**Comment by ulisigi** [comment, marked as answer]
<p>Hi <a class="mention" href="/sohels07">@sohels07</a>!</p>
<p>It looks like GitHub has nearly two dozen partnerships for their student development pack, but Udacity doesn’t appear to be one of them at this time. Check it out: <a href="https://education.github.com/pack" rel="nofollow noopener">https://education.github.com/pack</a></p>
<p>Hope this helps!</p>

**Comment by sohels07** [comment]
<p>Thanks <a class="mention" href="/ulisigi">@ulisigi</a>. </p>
<p>So, is there any nanodegree or any other course to learn android studio in this pack.</p>

**Comment by ulisigi** [comment]
<p>There doesn’t appear to be anything geared toward android studio in the GitHub student pack, but Udacity does offer Developing Android Apps, the first course of the nanodegree, for free: <a href="https://www.udacity.com/course/new-android-fundamentals--ud851" rel="nofollow noopener">https://www.udacity.com/course/new-android-fundamentals–ud851</a></p>

**Comment by sohels07** [comment]
<p>Thank you sir <a class="mention" href="/ulisigi">@ulisigi</a>. </p>

**Comment by nadiajoyce** [comment]
<p>Hi <a class="mention" href="/sohels07">@sohels07</a>,</p>
<p>This post was moved to a different board that fits your topic of discussion a bit better. This means you’ll get better engagement on your post, and it keeps our Community organized so users can more easily find information.</p>
<p>As you’ll notice, your Topic is now in the How to use Git and GitHub board. No action is needed on your part; you can continue the conversation as normal here.</p>
<p>Cheers!</p>
